tests/test_slogpy_file_info.py

```diff
@@ -0,0 +1,54 @@
+"""Import slogpy from slog"""
+import re
+import tempfile
+
+import pytest
+
+from slogpy.slog import Slog as slog  # noqa: N813
+
+"""This is a setup function"""
+def setupfunction():
+    """Initialize the logger with file logging enabled"""
+    # Create a temporary log file for testing
+    log_file = tempfile.mktemp(suffix=".log")
+    slog.initialize(file_logging=True, log_file=log_file)
+
+
+"""Unit test for logging"""
+@pytest.mark.parametrize("msg", [
+    # happy cases
+    ("This is a test for the file "),
+    ("try 2"),
+    # Testing with colors
+    ("try [red] 3"),
+    # testing nulls
+    (None),
+    # testing array of strings
+    (["String 1", "String 2", "String 3"])
+])
+def test_info_file_output(msg):
+    """Passing a message to the info method and then read the message from the log file"""
+    slog.info(msg)
+     # Accessing the log file attribute directly
+    log_file = slog._log_file
```

Comment:
@cloverspark I think we can extract out the file reading component to its own function to be used by both functions. 

Reason being if we need to add more tests or the way we read the contents of file change (I think this is the bigger reason, personally prefer to abstract I/O as much as possible), its a lot easier to update it.